halo2_backend/src/plonk/prover.rs

```diff
@@ -143,29 +151,33 @@ impl<
         pk: &'a ProvingKey<Scheme::Curve>,
         // TODO: If this was a vector the usage would be simpler.
         // https://github.com/privacy-scaling-explorations/halo2/issues/265
-        instances: &[&[&[Scheme::Scalar]]],
+        circuits_instances: &[&[&[Scheme::Scalar]]],
         rng: R,
         transcript: &'a mut T,
     ) -> Result<Self, Error>
     where
         Scheme::Scalar: WithSmallOrderMulGroup<3> + FromUniformBytes<64>,
     {
-        for instance in instances.iter() {
+        for instance in circuits_instances.iter() {
             if instance.len() != pk.vk.cs.num_instance_columns {
                 return Err(Error::InvalidInstances);
             }
         }
 
-        // Hash verification key into transcript
+        // Hash verification key into transcript [TRANSCRIPT-1]
         pk.vk.hash_into(transcript)?;
 
         let meta = &pk.vk.cs;
         let phases = meta.phases().collect();
 
         let domain = &pk.vk.domain;
 
+        // commit_instance_fn is a helper function to return the polynomials of instance columns while
```

Comment:
Suggestion: indicate that this helper function commits the polynomials of the instance columns.